sound/soc/sof/intel/hda.c

```diff
@@ -1368,6 +1449,7 @@ struct snd_soc_acpi_mach *hda_machine_select(struct snd_sof_dev *sdev)
 				return NULL;
 
 			sof_pdata->tplg_filename = tplg_filename;
+			dev_info(sdev->dev, "tplg fixup: %s\n", sof_pdata->tplg_filename);
```

Comment:
is this needed? I thought we already logged the topology file name, I see it in all CI results.

Reply:
I used this string for testing. Removed.